libs/core/include_local/include/hpx/local/mutex.hpp

```diff
@@ -13,5 +13,5 @@
 #include <hpx/thread_support/unlock_guard.hpp>
 
 namespace hpx {
-    using hpx::util::unlock_guard;
+    using hpx::unlock_guard;
 }    // namespace hpx
```

Comment:
This can go away.

Reply:
Done!

Reply:
That's not what I meant. You now already have `hpx::unlock_guard` defined, there is no need to reimport it into the HPX namespace again. You can remove the whole section (L15-17).

Reply:
yes, this is what I have done, please check here https://github.com/STEllAR-GROUP/hpx/pull/6086/commits/fc73f749c0a949e67e780dd3b9c003a7a18292e7 because this is the outdated version of the file

Reply:
Yah, sorry, I was looking at an older commit. All is well.